app/views/home/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if current_user %>
+  Logged in as <%= current_user.email %>.
+  <%= link_to "Log Out", logout_path %>
+<% else %>
+  <%= link_to "Sign Up", signup_path %> or
+  <%= link_to "Log In", login_path %>
+<% end %>
```

Comment:
why not add this to the header?
also dont we already have a home page as `startup`?